src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/PollingEserviceResponse.java

```diff
@@ -9,7 +11,7 @@
 
 @Data
 @Builder
-public class PollingEserviceResponse {
+public class PollingEserviceResponse implements Dto {
```

Comment:
Response implements Response interface not dto

Reply:
done and pushed